src/textOnlyPackages/src/microsoft.netcore.dotnethostpolicy/Microsoft.NETCore.DotNetHostPolicy.2.1.17.csproj

```diff
@@ -0,0 +1 @@
+<Project Sdk="Microsoft.NET.Sdk" />
```

Comment:
Looks like this csproj ended up in the wrong place? (No version)

Reply:
Thanks, fixed.